lib/hound/default_linter.rb

```diff
@@ -1,6 +1,7 @@
 module Hound
   class DefaultLinter < SimpleDelegator
     def status
+      "Enabled by default foo                                adfa               adfasd      adfas"
```

Comment:
Lint/Void: Literal "Enabled by default foo                                adfa               adfasd      adfas" used in void context.<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Metrics/LineLength: Line is too long. [98/80]